### DISCUSSION TITLE
PHP support for V2?

### DISCUSSION THREAD

**Discussion by stepwn** [WalletConnect/walletconnect-monorepo#2473] [Developer Support]
Hello, I currently have a repository, stepwn/LoopPress, that enables walletconnect for loopring supported token gating on wordpress websites. I am currently using the v1 libraries, and it seems they are going to be depreciated. I don't do any transaction signing, just public wallet verification.

Is there a route I can take to update my plugin? Multiple people are using it (because it does not require Node).

From what I have read, v2 seems to require Node, and losing walletconnect support might kill the project :(

I appreciate any responses.

**Comment by finessevanes** [comment]
@stepwn are you using wagmi?

**Reply by stepwn** [reply]
I was able to come up with a solution by building and packaging the relevant web3modal and walletconnect libraries. I believe this is the only solution for PHP until it is hosted on unpkg. 

check out my implementation at https://github.com/stepwn/LoopPress/ 

The bundled walletconnect library is in the assets/ directory.

Please let me know if there is anything critically wrong with my implimentation!

**Reply by stepwn** [reply]
I was using CDN before for v1, I just checked and found this page https://docs.walletconnect.com/2.0/web/walletConnectModal/installation#add-packages-cdn

it looks like they added CDN support since I last checked/made this post? If so I will update my repo

Edit* the page no longer exists since I first linked
